### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by Guy-Gustave**
a project build with ruby on rails implemented on basic of active record trying to build a muni_redit page.
we created 3 models, user post and comment.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,53 @@
...
+## 🔧 Built with<a name = "with"></a>
+  Ruby on Rails
 
-* Services (job queues, cache servers, search engines, etc.)
+  ## 🔨 Setup    <a name = "setup"></a>
+- Clone the project from github using
+- <code>$ https://github.com/Guy-Gustave/active_record/tree/feature_branch</code>
+- <code>cd active_record</code>
+- <code>$ bundle install</code>
+
```
Comment (christianotieno) [inline README.md]: # Changes required
- [ ] Kindly add instructions on how to do migrations upon cloning the project.

**Review by christianotieno — CHANGES_REQUESTED** [review]
## CHANGES REQUIRED ♻️ 
Hello @Guy-Gustave and @tacodtripe 👋 
- You guys have done a commendable job. Keep it up.
- Before we finish this up, please let's ensure we take care of the following small issue 👇 
##
### Christian Otieno (TSE)

**Comment by Guy-Gustave** [comment]
I changed the readme as required showing how to set up the migration.

**Review by wathigo — APPROVED** [review]
### Approved :heavy_check_mark: 
Great work on team :clap: 

**You may move on.**

**Simon.Wathigo.**

**Pull request merged**